src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/Cache.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
```

Comment:
This file is yours, should be copyright you, unless you work for CloudBees and I haven’t noticed ;-)

Reply:
Also 2017-2018 as you touched it both years

Reply:
Copy & paste ... And I do not care a lot about having my name in the license header ...

Reply:
Legally the MIT license is not applied if the copyright is incorrectly attributed. It would be better as your real name... but `bguerin` is probably ok.

Reply:
Also I cannot add copyright onto your work... I could add my copyright to my modifications of your work... but only if you have put copyright there in the first place